### PR TITLE
SDEV-4605 - remove input validation pattern check on chromosomeBand -…

### PR DESCRIPTION
**Pull request by dustinbleile**
… check for colon is non-standard.

Just remove the check for chromosomeband format.  It is only used as a comment anyway.

### CODE REVIEW

File: pori_python/ipr/content.spec.json [hunk 1]
```diff
@@ -99,8 +99,7 @@
             "items": {
                 "properties": {
                     "chromosomeBand": {
-                        "example": "X:p12.2",
-                        "pattern": "^(\\S+:\\S+?)?$",
+                        "example": "p12.2",
```
Comment (mathieulemieux) [inline pori_python/ipr/content.spec.json]: Is there a field for the chromosome name somewhere else?

Comment (dustinbleile) [reply pori_python/ipr/content.spec.json]: I do not think so.  This is only used as extra gene info.

It seems like standard pattern would be either
p12.2

or without the colon
Xp12.2

Comment (mathieulemieux) [reply pori_python/ipr/content.spec.json]: Eleanor and I are worried that the chromosome name might be usefull somehow. I know we briefly talked about it at the last meeting so sorry if I'm asking an already answered question, but why do we have to remove the chromosome prefix? Below is an example of a copyVariant record:
image

Comment (dustinbleile) [reply pori_python/ipr/content.spec.json]: Change the example to Xp12.2

If you want to keep the chromosome information is fine, but getting 20 thousand error messages about chromosome band regex check, that is wrong, is irritating.

In your example above, the proper chromosomeband should be 2p11.2

https://www.sciencedirect.com/topics/immunology-and-microbiology/chromosome-band#:~:text=Nomenclature%20for%20chromosome%20bands%20includes,short%20arm%2C%20band%2012.3).

Comment (dustinbleile) [reply pori_python/ipr/content.spec.json]: Could we change the description of 'cna' to something like - Copy Number Absolute, for just straight copy number?

I don't think anyone knows what an alteration ratio would be.
  
                  "cna": {
                        "description": "The copy number alteration (CNA) ratio",

Comment (mathieulemieux) [reply pori_python/ipr/content.spec.json]: > getting 20 thousand error messages about chromosome band regex check, that is wrong, is irritating.

I can imagine!
I'm totally fine with 'Xp12.2'. Maybe the colon was inspired by gkb/hgvs ways of describing variants. If ever needed, I suppose we can adapt downstream scripts to extract chromosome name from it.

Comment (mathieulemieux) [reply pori_python/ipr/content.spec.json]: > Could we change the description of 'cna' to something like - Copy Number Absolute, for just straight copy number?

If it's a better description, I don't see why not.

File: pori_python/ipr/content.spec.json [hunk 2]
```diff
@@ -99,12 +99,11 @@
...
                 "properties": {
                     "chromosomeBand": {
-                        "example": "X:p12.2",
-                        "pattern": "^(\\S+:\\S+?)?$",
+                        "example": "Xp12.2",
                         "type": "string"
                     },
                     "cna": {
-                        "description": "The copy number alteration (CNA) ratio",
+                        "description": "Copy Number, Absolute (cna)",
```
Comment (dustinbleile) [inline pori_python/ipr/content.spec.json:106]: With this description, it makes sense to use to use this field for the copy number tool output, unaltered.

So Purple can upload non-integer copy numbers and the older CNASEQ can still have its raw copy number alteration uploaded.

Comment (mathieulemieux) [reply pori_python/ipr/content.spec.json:106]: awesome. I suspected the new pipeline had something to do with it

**Review by mathieulemieux — APPROVED** [review]

**Pull request merged**